ussologin/ussologin.go

```diff
@@ -19,6 +18,12 @@ import (
 	"gopkg.in/macaroon-bakery.v1/httpbakery"
 )
 
+const (
+	EmailKey    = "Email"
```

Comment:
I'd unexport these and set them to the strings in #7 because that's what it says in the spec